### PR TITLE
DrupalNode to Node and drupal_node to node

### PR DESCRIPTION
**Pull request by aayushgupta1**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] all tests pass -- `rake test:all`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -237,10 +237,10 @@ def batch
       nodes = 0
       users = []
       params[:ids].split(',').uniq.each do |nid|
-        node = DrupalNode.find nid
-        node.spam
+        node1 = Node.find nid
```
Comment (ryzokuken) [inline app/controllers/admin_controller.rb]: is this really required? `node` and `Node` are still different.

Comment (aayushgupta1) [inline app/controllers/admin_controller.rb]: It's not needed. It is just to be on a safer side.

Comment (aayushgupta1) [inline app/controllers/admin_controller.rb]: Could you help me with the conflicts?

**Comment by aayushgupta1** [comment]
@jywarren @ryzokuken @mridulnagpal could you help me with this?

**Comment by ryzokuken** [comment]
@aayushgupta1 "The conflicts are too complex to resolve in the web editor"

Are you sure you have rebased your changes on top of the latest master?

**Comment by aayushgupta1** [comment]
@ryzokuken Kindly look at #1320 . There are no conflicts there.

**Pull request closed without merging**